OrbitLinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -2,6 +2,40 @@
 
 namespace LinuxTracing {
 
+void UprobesFunctionCallManager::ProcessUprobes(pid_t tid,
+                                                uint64_t function_address,
+                                                uint64_t begin_timestamp) {
+  auto& tid_timer_stack = tid_timer_stacks_[tid];
+  tid_timer_stack.emplace(function_address, begin_timestamp);
+}
+
+std::optional<FunctionCall> UprobesFunctionCallManager::ProcessUretprobes(
+    pid_t tid, uint64_t end_timestamp) {
+  if (tid_timer_stacks_.count(tid)) {
```

Comment:
Would be easier to read if we "early out" here. Then you don't have to indent most of the function body. 

if (tid_timer_stacks.count(tid) == 0) {
  return std::optional<FunctionCall>();
}

// Now we know what tid_timer_stacks.count(tid) > 0

auto& tid_timer_stack = ...